fixed overflow of row and made card width fixed in css

bprathamesh20 opened the pull request:
### PR Description:

Issue: Resolved Overflow and Fixed Card Width in CSS

### Changes Made:

-Addressed the issue of overflowing content within rows.
-Implemented a fixed width for cards to ensure consistent display.

**Before fix** 
<img width="960" alt="image" src="https://github.com/Manishak798/Dive-to-Dev_Hacktoberfest23/assets/98270118/ba5f6e40-ce06-4818-9256-38af4f9d79bb">


**After Fix**
<img width="960" alt="image" src="https://github.com/Manishak798/Dive-to-Dev_Hacktoberfest23/assets/98270118/69193021-5aa3-478d-a7fb-d041b65e5046">

bprathamesh20 committed:
fixed overflow of row and made card width fixed in css

Manishak798 reviewed (changes requested):
I want you to make changes to the card arrangement, Make a row with four cards.

bprathamesh20 committed:
added four cards to a row and fixed the overflow

bprathamesh20 commented:
### Changes Made:

- Adjusted the HTML structure and Bootstrap classes to ensure that all project cards maintain a uniform width.
- Removed the fixed width from individual cards to prevent horizontal overflow.
- Updated the custom CSS to control card spacing within a row.

![image](https://github.com/Manishak798/Dive-to-Dev_Hacktoberfest23/assets/98270118/860772f3-bb99-4bb8-a5ea-f033480d8739)

Manishak798 committed:
Merge branch 'main' into fixed-row-overflow
